src/pages/Unlock/Unlock.tsx

```diff
@@ -74,16 +91,18 @@ export const Unlock = () => {
             />
             <XaliasLoginWrapper {...commonProps} />
             <WebWalletLoginWrapper {...commonProps} />
-            <IframeButton
-              loginButtonText='Passkey Proxy'
-              loginType='passkey'
-              {...commonProps}
-            />
+            {isMobile && (
```

Comment:
why only on mobile ?

Reply:
it creates different addresses between phone and desktop and we decided to allow it only on mobile